controllers/linstorsatellite_controller.go

```diff
@@ -204,40 +200,22 @@ func (r *LinstorSatelliteReconciler) kustomizeNodeResources(lsatellite *piraeusi
 			secretName = lsatellite.Name + "-tls"
 		}
 
-		tlsPatches, err := TLSPatches(secretName)
+		p, err := SatelliteLinstorInternalTLSPatch(secretName)
 		if err != nil {
 			return nil, err
 		}
 
-		patches = append(patches, tlsPatches...)
+		patches = append(patches, p...)
 
 		if lsatellite.Spec.InternalTLS.CertManager != nil {
 			resourceDirs = append(resourceDirs, "pod/cert-manager")
 
-			certPatch, err := utils.ToEncodedPatch(&kusttypes.Selector{
-				ResId: resid.NewResId(resid.NewGvk("cert-manager.io", "v1", "Certificate"), "tls"),
-			}, []utils.JsonPatch{
-				{
-					Op:    utils.Replace,
-					Path:  "/spec/issuerRef",
-					Value: lsatellite.Spec.InternalTLS.CertManager,
-				},
-				{
-					Op:    utils.Replace,
-					Path:  "/metadata/name",
-					Value: secretName,
-				},
-				{
-					Op:    utils.Replace,
-					Path:  "/spec/secretName",
-					Value: secretName,
-				},
-			})
+			p, err := SatelliteLinstorInternalTLSCertManagerPatch(secretName, lsatellite.Spec.InternalTLS.CertManager)
```

Comment:
I don't quite see the benefit of converting these `JsonPatch` ones. They use a different format to the yaml resources anyway.

Reply:
You are right. And right now I don't know why they are `JsonPatch` in the first place.

Reply:
Replaced those patches with strategic merge ones. I only  used it because I thought I needed to change the names, but that is either:
* better done using `replacements`
* just not needed